auslib/log.py

```diff
@@ -26,42 +26,118 @@ def makeRecord(self, name, level, fn, lno, msg, args, exc_info, func=None, extra
                 # Request object, whose id is actually what we want.
                 # Without this we end up with the id of the proxy object, which
                 # is static for the life of the application.
+                # TODO: this doesn't seem to be 100% unique.
+                # Sometimes requests that happen around the same time will
+                # end up with the same value here. Possibly only when the
+                # query strings are the same, or srcip?
                 requestid = id(request._get_current_object())
```

Comment:
I might be missing something, but I'm not sure how that helps? If I generate a new random number for each call to make_record, each log entry (even from within the same request) will end up with a different id. I don't want to block on this in any case...I just didn't want to forget that this needs to get fixed at some point.